core/src/main/scala/org/locationtech/rasterframes/util/package.scala

```diff
@@ -203,6 +203,25 @@ package object util {
         .mkString("| ", " |\n| ", " |")
       header + body
     }
+
+    def toHTML(numRows: Int = 5, truncate: Boolean = false): String = {
+      import df.sqlContext.implicits._
+      val cols = df.columns
+      val header = "<thead>\n" + cols.mkString("<tr><th>", "</th><th>", "</th></tr>\n") + "</thead>\n"
+      val stringifiers = cols
```

Comment:
it may also make sense to have some truncation parameter /  check on the number of Columns? In some ML applications there can be hundreds of columns which are not going to be meaningful to look at. 

As below if truncating need to give some kind of message / visual indication. Pandas does this well.

Reply:
You'll have to explain to me what kind of assumptions you can make about what's meaningful or not.